source/git/gitsource.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/moby/buildkit/source"
 	"github.com/moby/buildkit/util/bklog"
 	"github.com/moby/buildkit/util/progress/logs"
+	"github.com/moby/buildkit/util/urlutil"
```

Comment:
would be good if you could put this change to separate commit but not a huge issue

Reply:
sure

Reply:
https://github.com/moby/buildkit/pull/2363